### PR TITLE
Ps updating validation

### PR DESCRIPTION
**Pull request by psantos1113**
Changes to use the builders in hub-common.

@ekerwin @jrichardBD @stevebillings

### CODE REVIEW

**Comment by stevebillings** [comment]
Looks good

File: server/src/main/java/com/blackducksoftware/integration/hub/teamcity/server/global/HubGlobalServerConfigController.java [hunk 1]
```diff
@@ -22,23 +22,12 @@
 package com.blackducksoftware.integration.hub.teamcity.server.global;
```
Comment (ekerwin) [inline server/src/main/java/com/blackducksoftware/integration/hub/teamcity/server/global/HubGlobalServerConfigController.java:22]: I feel like THIS class makes a compelling case for the validation changes - look at all that RED!!! :)

**Comment by ekerwin** [comment]
Great stuff!

**Comment by jamesrichard91** [comment]
Looks good to me

**Pull request merged**